posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -383,6 +402,161 @@ async def test_insert_into_s3_activity_puts_data_into_s3(
     )
 
 
+@pytest.mark.parametrize("compression", [None, "gzip", "brotli"], indirect=True)
+@pytest.mark.parametrize("model", [BatchExportModel(name="events", schema=None)])
+@pytest.mark.parametrize("file_format", FILE_FORMAT_EXTENSIONS.keys())
+# Use 0 to test that the file is not split up and 6MB since this is slightly
+# larger than the default 5MB chunk size for multipart uploads.
+@pytest.mark.parametrize("max_file_size_mb", [None, 6])
+async def test_insert_into_s3_activity_puts_splitted_files_into_s3(
+    clickhouse_client,
+    bucket_name,
+    minio_client,
+    activity_environment,
+    compression,
+    max_file_size_mb,
+    exclude_events,
+    file_format,
+    data_interval_start,
+    data_interval_end,
+    model: BatchExportModel | BatchExportSchema | None,
```

Comment:
Unnecessary to have the three types since we only parametrize this with `BatchExportModel(name="events", schema=None)`.

Reply:
Saves you from doing that if-statement below to resolve each of the types.